source/connection-string/connection-string-spec.rst

```diff
@@ -180,9 +180,9 @@ The values in connection options MUST be URL decoded by the parser. The values c
 - Strings:
     The value
 - Integer:
-    The value parsed as a integer
+  The value parsed as a integer. If the value is the empty string, then a WARN level logging message MUST be issued and the key MUST be ignored.
 - Boolean:
-    "true" and "false" strings MUST be supported.
+  "true" and "false" strings MUST be supported. If the value is the empty string, then a WARN level logging message MUST be issued and the key is ignored. Drivers that cannot log warning messages or where warnings might be easily overlooked MUST raise an error.
```

Comment:
I think "MAY raise an error" is the appropriate language here since it refers to a permitted deviation.

Reply:
I have updated the language.